apps/davi/public/locales/es/translation.json

```diff
@@ -0,0 +1,435 @@
+      "function": "Función",
+      "functionName": "Nombre de la Función",
+      "functionSignature": "Firma de la Función",
+      "assetTransfer": "Transferencia de Activo",
+      "functionCall": "Llamada a la Función",
+      "functionCalls": "Llamadas a la Función",
+      "noPermissionsSet": "No hay permisos establecidos",
+      "dataNotAvailable": "Datos de permisos no disponibles",
+      "allowedAmount": "Cantidad permitida"
+    },
+    "transfer": {
+      "transfer": "Tranferir",
+      "unknownToken": "Token desconocido"
+    },
+    "repMint": {
+      "mint": "Acuñar",
+      "mintRep": "Acuñar REP",
+      "recipient": "Destinatario",
+      "recipientTooltip": "La dirección que va a recibir el REP acuñado.",
+      "repPercent": "Reputación en %",
+      "repPercentTooltip": "El porcentaje de la reputación total suministrada que será acuñada.",
+      "repAmount": "Cantidad de Reputación",
+      "repAmountTooltip": "La cantidad de REP que será acuñado."
+    },
+    "ens": {
+      "updateContentCoreAction": "Actualizar Contenido ENS",
+      "nameTooltip": "El nombre del nombre ENS que será registrado.",
+      "ipfsHashToolTip": "El hash del IPFS que será establecido",
+      "domain": "Dominio",
+      "currentContent": "Contenido actual",
+      "newContent": "Nuevo contenido",
+      "updateContent": "ENS: Actualizar Contenido",
+      "name": "Nombre ENS",
+      "ipfsHash": "Hash CID/IPFS",
+      "none": "Ninguno",
+      "expandLinesOfCode": "Expandir {{numLines}} líneas de código",
+      "validation": {
+        "nameCannotBeEmpty": "El nombre no puede estar vacío",
+        "domainNameInvalidLength": "Los nombres del dominio tienen una lóngitud inválida",
+        "domainNameCannotIncludeSpaces": "El nombre del dominio no puede incluir espacios",
+        "domainCannotBeMoreThanThreeLevels": "El dominio no puede tener más de tres niveles de profundidad (subdominio.dominio.eth)",
+        "ipfsHashNotValid": "El hash del IPFS no es válido",
+        "onlyCidV0Supported": "Únicamente se admite CID v0"
+      }
+    },
+    "approval": {
+      "approve": "Aprobar",
+      "spender": "gastador",
+      "selectTokensToSpend": "Seleccionar token a gastar",
+      "approveSpendingCall": "Aprobar la llamada de gasto",
+      "addressToWhichTheExpenseIsBeingAuthorized": "Dirección a la cual la transacción está siendo autorizada",
+      "amountBeingApproved": "Cantidad siendo aprobada",
+      "approveTokenSpending": "Aprobar el gasto de {{tokenSymbol}} token"
+    },
+    "config": {
+      "setGuildConfig": "Establecer Configuración de Gremio",
+      "noValueHasBeenUpdatedFromCurrentGuildConfig": "Ningún valor ha sido actualizado de la configuración actual de gremio",
+      "proposalTimeHasToBeMoreThanZero": "El tiempo de Propuesta tiene que ser más de 0",
+      "lockTimeHasToBeHigherOrEqualToProposalTime": "El tiempo de Bloqueo tiene que ser mayor o igual al tiempo de Propuesta",
```

Comment:
@rossneilson  For this form we are not using labels translations for each field `apps/davi/src/components/ActionsBuilder/SupportedActions/SetGuildConfig/constants.ts`
This is referring to specific fields in `setConfig` function. I am wondering if we should translate this without doing the translation for each field name, like: "lockTime tiene que ser mayor o igual a proposalTime" since that are the labels the user will see on the UI. 
Or we should also add translations for this setGuildConfig component in actionsbuilder and use translations for setGuildConfig form and continue with this pr translation as it is (I think this is the best option)

Reply:
Yeah lets go with adding translations here and we'll fix the action builder later. We also have to do internationalisation in rich contracts v2 soon.
(no changes required from you here @dianneriivera)